core/vm/fvm/stub.cpp

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "vm/fvm/stub.hpp"
+
+namespace fc::vm::fvm::stub {
+  decltype(callbacks) callbacks;
+
+  extern "C" FvmError cgo_blockstore_get(FvmMachineId machine_id,
+                                         const uint8_t *key,
+                                         int32_t key_size,
+                                         uint8_t **out_value,
+                                         int32_t *out_value_size) {
+    if (!callbacks.ipldGet) {
+      return FvmError::kInvalidHandle;
+    }
+    if (!out_value) {
+      return FvmError::kInvalidArgument;
+    }
+    if (!out_value_size) {
+      return FvmError::kInvalidArgument;
+    }
+    Bytes value;
+    const auto res{callbacks.ipldGet(machine_id, {key, key_size}, &value)};
+    if (res == FvmError::kOk) {
+      *out_value = gsl::narrow<uint8_t *>(malloc(value.size()));
+      memcpy(*out_value, value.data(), value.size());
+      *out_value_size = value.size();
```

Comment:
Can we move result?

Reply:
No, `vector` is not compatible with `malloc` required by ffi